From a0b8462afc7cf3ae0b58bd62b2accb4db7ca88ef Mon Sep 17 00:00:00 2001
From: yunjie <yunjie.jia@nxp.com>
Date: Mon, 6 Mar 2023 11:21:08 +0000
Subject: [PATCH] MA-21101 [i.MX 93_EVK] enable SDIO-UART IW612 module for WiFi
 and BT

Reason: add IW612 entry to wifi_mod_para.conf
Tested: on i.MX93

Change-Id: I533f55334766c1ac9207843df2dcfb94715381ca
Signed-off-by: yunjie <yunjie.jia@nxp.com>
---
 nxp/android_wifi_mod_para.conf | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/nxp/android_wifi_mod_para.conf b/nxp/android_wifi_mod_para.conf
index 6183243..a86c895 100644
--- a/nxp/android_wifi_mod_para.conf
+++ b/nxp/android_wifi_mod_para.conf
@@ -27,6 +27,24 @@
 # this configuration file, and set the corresponding value
 # according to your real needs
 
+SDIW612 = {
+        cfg80211_wext=0xf
+        sta_name=wlan
+        uap_name=wlan
+        wfd_name=p2p
+        max_vir_bss=1
+        cal_data_cfg=none
+        ps_mode=2
+        auto_ds=2
+        fw_name=sduart_nw61x_v1.bin.se
+        fw_serial=0
+        cntry_txpwr=2
+        country_ie_ignore=1
+        beacon_hints=1
+        slew_rate=2
+        host_mlme=1
+}
+
 SD8997 = {
 	cfg80211_wext=0xf
 	sta_name=wlan
-- 
2.34.1
